parts/k8s/kubernetescustomscript.sh

```diff
@@ -51,20 +51,17 @@ else
     FULL_INSTALL_REQUIRED=true
 fi
 
-holdWALinuxAgent() {
-    if [[ $OS == $UBUNTU_OS_NAME ]]; then
-        wait_for_apt_locks
-        retrycmd_if_failure 120 5 25 apt-mark hold walinuxagent || exit $ERR_HOLD_WALINUXAGENT
-        wait_for_apt_locks
-    fi
-}
+if [[ $OS == $UBUNTU_OS_NAME ]]; then
+    wait_for_apt_locks
+    retrycmd_if_failure 120 5 25 apt-mark hold walinuxagent || exit $ERR_HOLD_WALINUXAGENT
+    wait_for_apt_locks
+fi
 
 if [[ ! -z "${MASTER_NODE}" ]]; then
     installEtcd
 fi
 
 if $FULL_INSTALL_REQUIRED; then
-    holdWALinuxAgent
```

Comment:
instead of remove the holdWALinuxAgent function, maybe just move L67 before L66, will that make future re-enable/merge easier?